src/control/server/commands.go

```diff
@@ -155,17 +155,29 @@ func (p *PrepNvmeCmd) Execute(args []string) error {
 // configuring in AppDirect mode and creating relevant namespaces.
 type PrepScmCmd struct {
 	Reset bool `short:"r" long:"reset" description:"Reset modules to memory mode after removing namespaces"`
+	Force bool `short:"f" long:"force" description:"Perform format without prompting for confirmation"`
 }
 
 // Execute is run when PrepScmCmd activates
 //
 // Perform task then exit immediately. No config parsing performed.
-func (p *PrepScmCmd) Execute(args []string) error {
+func (p *PrepScmCmd) Execute(args []string) (err error) {
+	needsReboot := false
+	pmemDevs := make([]pmemDev, 0)
+
 	ok, _ := common.CheckSudo()
 	if !ok {
 		return errors.New("subcommand must be run as root or sudo")
 	}
 
+	log.Info("Memory allocation goals for SCM will be changed and namespaces " +
+		"modified, this could be a destructive operation. Please " +
```

Comment:
IMO "this could be a destructive operation" is an understatement.

Reply:
"this will be a destructive operation" better?

Reply:
Sounds good to me.